.travis.yml

```diff
@@ -1,6 +1,7 @@
 sudo: false
 script:
   - curl https://raw.githubusercontent.com/shapesecurity/CLA/master/cla-travis.sh | bash
+  - npm i
```

Comment:
`npm i` is a nice shorthand when using the CLI. When writing down commands in scripts, use long flag names and full subcommand names. `npm install`.